modules/client-configuration/pages/clients-oracle.adoc

```diff
@@ -72,7 +88,7 @@ The channels you need for this procedure are:
 
 |===
 
-include::snippets/add_channels_cli.adoc[]
+include::snippets/add_channels_scc.adoc[]
```

Comment:
We don't use SCC for Oracle on Uyuni.

Reply:
![image](https://user-images.githubusercontent.com/3914967/98321397-b315c700-2030-11eb-808a-76496318ed95.png)